api/git/azure.go

```diff
@@ -136,21 +136,19 @@ func (a *azureDownloader) latestCommitID(ctx context.Context, options fetchOptio
 
 	var refs struct {
```

Comment:
As we no longer pulling references, it would make sense to rename `buildRefsUrl` to correspond the api call and change name of this struct accordingly.

Reply:
yes, will do.